scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml

```diff
@@ -0,0 +1,504 @@
+          host:
+            start: 10.162.67.51
+            end: 10.162.67.100
+      nova_fixed:
+        vlan: 717
+        subnet: 44.0.0.0
+        netmask: 255.255.0.0
+        broadcast: 44.0.255.255
+        router: 44.0.0.1
+        ranges:
+          router:
+            start: 44.0.0.1
+            end: 44.0.0.10
+          dhcp:
+            start: 44.0.0.11
+            end: 44.0.255.250
+      nova_floating:
+        vlan: 715
+        subnet: 10.162.67.0
+        netmask: 255.255.255.0
+        broadcast: 10.162.67.255
+        ranges:
+          host:
+            start: 10.162.67.101
+            end: 10.162.67.200
+      bmc:
+        router: 10.162.65.254
+        vlan: 711
+        use_vlan: true
+        subnet: 10.162.65.0
+        broadcast: 10.162.65.255
+        ranges:
+          host:
+            start: 10.162.65.49
+            end: 10.162.65.89
+      bmc_vlan:
+        router: 10.162.65.254
+        vlan: 711
+        subnet: 10.162.65.0
+        broadcast: 10.162.65.255
+        ranges:
+          host:
+            start: 10.162.65.149
+            end: 10.162.65.199
+      os_sdn:
+        vlan: 716
+      admin:
+        vlan: 714
+        subnet: 10.162.66.0
+        netmask: 255.255.255.128
+        broadcast: 10.162.66.127
+        router: 10.162.66.1
+        ranges:
+          admin:
+            start: 10.162.66.10
+            end: 10.162.66.22
+          dhcp:
+            start: 10.162.66.30
+            end: 10.162.66.49
+          host:
+            start: 10.162.66.50
+            end: 10.162.66.69
+          switch:
+            start: 10.162.66.23
+            end: 10.162.66.29
+  deployment:
+    elements:
+      switch_config:
+      - crowbar.qa4.cloud.suse.de
+      network:
+      - crowbar.qa4.cloud.suse.de
+      - "@@service1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      - "@@controller1@@"
+      - "@@compute1@@"
+      - "@@compute2@@"
+- barclamp: dns
+  attributes:
+    domain: qa4.cloud.suse.de
+    forwarders:
+    - 10.162.66.1
+    records:
+      multi-dns:
+        ips:
+        - 10.11.12.13
+  deployment:
+    elements:
+      dns-server:
+      - crowbar.qa4.cloud.suse.de
+      dns-client:
+      - crowbar.qa4.cloud.suse.de
+      - "@@service1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      - "@@controller1@@"
+      - "@@compute1@@"
+      - "@@compute2@@"
+- barclamp: logging
+  attributes:
+  deployment:
+    elements:
+      logging-server:
+      - crowbar.qa4.cloud.suse.de
+      logging-client:
+      - "@@service1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      - "@@controller1@@"
+      - "@@compute1@@"
+      - "@@compute2@@"
+- barclamp: ntp
+  attributes:
+  deployment:
+    elements:
+      ntp-server:
+      - crowbar.qa4.cloud.suse.de
+      ntp-client:
+      - "@@service1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      - "@@controller1@@"
+      - "@@compute1@@"
+      - "@@compute2@@"
```

Comment:
Can we rely on default values for the barclamps above or are all the configs needed for some reason?